examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/PaywallTesterApp.kt

```diff
@@ -31,15 +32,26 @@ private fun AppNavHost(
         modifier = modifier,
     ) {
         composable(AppScreen.Main.route) {
-            MainScreen(navigateToPaywallScreen = { offering ->
-                navController.navigate(AppScreen.Paywall.route.plus("/${offering?.identifier}"))
-            })
+            MainScreen(
+                navigateToPaywallScreen = { offering ->
+                    navController.navigate(AppScreen.Paywall.route.plus("/${offering?.identifier}"))
+                },
+                navigateToPaywallFooterScreen = { offering ->
```

Comment:
Note that I've made showing the paywall as a footer a top level destination in the navhost, same as the normal navigation one. We might want to think about adding another composable to display the PaywallFooter as a dialog, but that can come later.